migrate/starter/starter.go

```diff
@@ -269,6 +275,8 @@ func (v *value) Set(s string) error {
 	return nil
 }
 
+func (v *value) IsBoolFlag() bool { return true }
```

Comment:
When this func is set, value is not necessary to be set for the flag(`--version` is accepted too). It helps to parse bool flags correctly.